src/print.rs

```diff
@@ -26,6 +26,7 @@ const DEFAULT_PROMPT_ORDER: &[&str] = &[
     "golang",
     "java",
     "nix_shell",
+    "elixir",
```

Comment:
Looks like you caught all the places this needs to be added in the code. Well done!